src/mapOfPeople.js

```diff
@@ -40,7 +40,14 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const mapPeople = new Map();
+
+  people.forEach((item, index) => !mapPeople.has(item)
```

Comment:
краще йти від `true`, а не `false`
```suggestion
  people.forEach((item, index) => mapPeople.has(item)
```